steps/plan/outputs.schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "plan": {
+      "type": "object",
```

Comment:
I think this should just be `"type": "string"` because by the time anything that consumes this sees it it will have been converted from this interchange format